virtio_gpu.c

```diff
@@ -17,6 +17,7 @@
 #include "util.h"
 #include "virgl_hw.h"
 #include "virtgpu_drm.h"
+#include <i915_drm.h>
```

Comment:
currently we use i915 to allocate scanout buffer for virtio-gpu, is this patch still needed?